shamil_latypov/graph_printer.cpp

```diff
@@ -70,8 +70,8 @@ std::string GraphPrinter::print_edge(const Edge& edge) const {
   std::stringstream ss;
   ss << "{\n      \"id\": " << edge.get_id() << ",\n      \"vertex_ids\": ["
      << edge.get_vertex1_id() << ", " << edge.get_vertex2_id()
-     << "],\n      \"color\": \"" << color_to_string(edge.get_color())
-     << "\"\n    }";
+     << "],\n      \"color\": \""
+     << GraphPrinter::print_edge_color(edge.get_color()) << "\"\n    }";
```

Comment:
Внутри объекта нет необходимости писать его префикс
```suggestion
     << print_edge_color(edge.get_color()) << "\"\n    }";
```